Conductor: explain `COMPOSER_SKIP_SCRIPTS` (#125)
packagist/private-packagist-docs: docs/conductor/getting-started.md

Once the workflow is added to your GitHub repository, Conductor needs to verify your CI setup before you can start receiving PRs.

### Skip Composer scripts

If you've configured Composer to run scripts for certain events that make changes which are irrelevant for the Conductor PR, you can skip 
these script handlers by setting the [COMPOSER_SKIP_SCRIPTS](https://getcomposer.org/doc/03-cli.md#composer-skip-scripts) environment variable.

> For example, if you use `npm` to build static assets when the `post-install-cmd` command event is triggered, update your GitHub action like this:

Reviewer: ```suggestion
For example, if you use `npm` to build static assets in your `post-install-cmd` script you can update your GitHub action to skip it:
```

Reviewer: but you can keep it as is. Was thinking maybe it doesnt need the example at alL :D

Author: Thanks, I like your suggestion better, reads a bit easier.
